wcc-contentful-app/app/views/sections/_testimonials.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="row">
+  <div class="column">
+    <% section.testimonials&.each do |testimonial|
+      next unless testimonial %>
```

Comment:
Not exactly sure why this is necessary? Is it just to handle the scenario where a testimonial could possibly be nil?

Reply:
Yes - anytime you have a broken link in an array of links, it comes across as nil.  If they unpublish a testimonial without removing it from the testimonial section, we don't want to blow up.